parsing/ast_invariants.ml

```diff
@@ -117,7 +117,7 @@ let iterator =
   in
   let open_description self opn =
     super.open_description self opn;
-    simple_longident opn.popen_lid
+    super.module_expr self opn.popen_expr
```

Comment:
Is this call to super.module_expr really necessary?  It should be done already by super.open_description, no?

Reply:
Good catch. Fixed.